src/utils/constants.py

```diff
@@ -36,11 +36,8 @@
 TO_SECONDS = 1000
 
 # GENERAL SIZES
-# 8 x 8 pixels
-SMALL_SIZE = (30, 30)
-
 # 16 x 16 pixels
-GENERAL_SIZE = (60, 60)
+GENERAL_SIZE = (50, 50)
```

Comment:
The idea is to keep 60 x 60, because this is the designated size for the screen, since 15 blocks high must fit on the screen in order to maintain the positions of the elements.

Reply:
The JSON configuration work was done previously before the sizes, doing it again is double effort.

Reply:
All boxes within the JSON were set to 50x50